spec.bs

```diff
@@ -371,13 +371,11 @@ To <dfn>register an import map</dfn> given an {{HTMLScriptElement}} |element|:
 <h2 id="resolving">Resolving module specifiers</h2>
 
 <div class="note">
-During [=resolve a module specifier|resolving a module specifier=], the following algorithms check candidate entries of [=specifier maps=], from most-specific to least-specific scopes and to top-level "`imports`", from most-specific to least-specific prefixes.
-For each entry, the result is one of the following:
+  During [=resolve a module specifier|resolving a module specifier=], the following algorithms check candidate entries of [=specifier maps=], from most-specific to least-specific scopes (falling back to top-level "`imports`"), and from most-specific to least-specific prefixes. For each candidate, the result is one of the following:
 
-  - Successfully resolves a specifier to a [=URL=].
-    This makes the [=resolve a module specifier=] algorithm return immediately successfully with the [=URL=].
-  - Throws an error.
-    This makes the [=resolve a module specifier=] algorithm fail immediately, i.e. throw the error, without any further fallbacks.
+  - Successfully resolves a specifier to a [=URL=]. This makes the [=resolve a module specifier=] algorithm immediately return that [=URL=].
+  - Throws an error. This makes the [=resolve a module specifier=] algorithm rethrow the error, without any further fallbacks.
+  - Fail to resolve, without an error. In this case the algorithm moves on to the next candidate.
```

Comment:
nit: I'm wondering better labels...
IIUC the third case corresponds to Step 1.3 of #resolve-an-imports-match, right?
"Throws an error" and "Fail to resolve" sound similarly and thus the difference between them might be unclear.
Alternatives I came up with are:

- "Throws an error" and "Continue"
- "Fail to resolve, throwing an error" and "Fail to resolve, without throwing an error"
  (Step 1.3 is just handling non-matching specifier keys, which doesn't look like failing something though)

Any idea?

Reply:
I was thinking it it corresponds to both step 1.3 and step 2.

Hmm maybe

- Resolves the specifier to null. This terminates the resolve a module specifier algorithm with a thrown exception.
- Does not resolve the specifier. In this case the algorithm moves on to the next candidate.

?

Reply:
"Resolves the specifier to URL", "Resolves the specifier to null" and "Does not resolve the specifier" looked great to me in terms of symmetry, but on second thought "Resolves to null" might be confusing with "Return null" in `resolve an imports match` (which is actually means "Does not resolve"), and might be a little inconsistent with `resolve a module specifier` interface (which throws exceptions rather than returning null).

"Does not resolve the specifier" still looks good to me, so how about something like:

- "Resolves the specifier to URL", "Throws an error" and "Does not resolve the specifier"
- "Resolves the specifier to URL", "Resolves the specifier to an error" and "Does not resolve the specifier"

?